libriscan/biblios/static/js/word_selector.js

```diff
@@ -38,30 +38,50 @@ class WordSelector {
       print_control: wordBlock.dataset.wordPrintControl,
       extraction_id: wordBlock.dataset.wordExtractionId,
       suggestions: this._parseSuggestions(wordBlock.dataset.wordSuggestions),
-      geometry: {
-        x0: parseFloat(wordBlock.dataset.wordGeoX0),
-        y0: parseFloat(wordBlock.dataset.wordGeoY0),
-        x1: parseFloat(wordBlock.dataset.wordGeoX1),
-        y1: parseFloat(wordBlock.dataset.wordGeoY1)
-      }
+      // Limits for geometry coordinates removed from data-words-json
```

Comment:
Going to comment/remove returning these geo coordinates unless we can use them for perhaps image bounding boxes.
Should improve HTML rendering performance and reduce HTML clutter while inspecting.